Makefile

```diff
@@ -354,4 +356,6 @@ helm-publish:
 
 	helm repo index $(HELM_LOCAL_REPO) --url $(HELM_REPOSITORY) --merge $(HELM_LOCAL_REPO)/index.yaml
 	gsutil rsync -d $(HELM_LOCAL_REPO) $(HELM_BUCKET)
+
+	gsutil setmeta -h "Content-Type:text/yaml" -h "Cache-Control: $(HELM_MANIFEST_CACHE_CONTROL)" $(HELM_BUCKET)/index.yaml
```

Comment:
tiny nit: for bash safety, single quotes (`'`) are preferred unless you need to evaluate a variables inside